sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -77,6 +77,21 @@ export interface CertificateClientInterface {
   ): Promise<DeletedCertificate>;
 }
 
+/**
+ * The latest supported KeyVault service API version
+ */
+export const LATEST_API_VERSION = "7.1-preview";
```

Comment:
How do people discover this? As discussed in the meeting yesterday, this may not be useful. .NET doesn't expose ours (its internal for other purposes).

Reply:
This is not exposed. In that case, is it ok to have it as a constant?

Reply:
Yeah. So the export is just to another module higher up the stack but not public?